pybb/views.py

```diff
@@ -443,6 +445,16 @@ def dispatch(self, request, *args, **kwargs):
                     return HttpResponse(self.quote)
         return super(AddPostView, self).dispatch(request, *args, **kwargs)
 
+    def post(self, request, *args, **kwargs):
+        response = super(AddPostView, self).post(request, *args, **kwargs)
+
+        if self.object and self.topic:
+            topic_updated.send(Post, post=self.object, request=self.request)
+            if not defaults.PYBB_DISABLE_SUBSCRIPTIONS and util.get_pybb_profile(self.object.user).autosubscribe and \
```

Comment:
I'm not a big fan to move the "auto subscribe" feature in the view. Why don't you let it in `post_saved` in `signals.py` ? If I want to have a custom own View (which does not inherit from this one), I'll have to re-code this pybb feature and maintain my own code with pybb evolves.
IMHO, we should keep this feature inside the `post_saved` called by the signal.